src/acc/opencl/smm/tune_multiply.py

```diff
@@ -152,12 +152,10 @@ def merge_into_csv(self, filenames):
                             print(
                                 "Worse result "
                                 + ifilename
-                                + " ignored when merging CSV file"
+                                + " ignored when merging CSV-file"
                             )
-                    except KeyError:
-                        print(
-                            "Malformed " + ifilename + " ignored when merging CSV file"
-                        )
+                    except (json.JSONDecodeError, KeyError):
+                        print("Failed to merge " + ifilename + " into CSV-file")
```

Comment:
the `pass` afterwards is therefore not needed anymore (Python does not need an explicit "ignore this exception")

Reply:
Good to know, thank you!